packages/network/src/initCache.ts

```diff
@@ -1,6 +1,9 @@
 import { arrayToIterator, deferred, mergeIterators, transformIterator } from "@latticexyz/utils";
+import fakeIndexedDb from "fake-indexeddb";
```

Comment:
I wonder if there's a way to update the methods below to use an async import so we don't have to import this into browser bundles.

```
const indexedDB = typeof self === 'undefined' ? await import('fake-indexeddb') : self.indexedDB.
```